src/main/java/org/sagebionetworks/web/server/servlet/filter/CrawlFilter.java

```diff
@@ -6,10 +6,9 @@
 import com.google.gwt.thirdparty.guava.common.base.Suppliers;
```

Comment:
Change the CrawlFilter so that it isn't an independent filter.  It is used by the HtmlInjectionFilter, when a bot has been detected